RecoTracker/LSTCore/src/alpaka/Quintuplet.h

```diff
@@ -0,0 +1,3237 @@
+                                                    segmentsInGPU,
+                                                    tripletsInGPU,
+                                                    lowerModule1,
+                                                    lowerModule2,
+                                                    lowerModule3,
+                                                    lowerModule4,
+                                                    lowerModule5,
+                                                    innerTripletIndex,
+                                                    outerTripletIndex,
+                                                    innerRadius,
+                                                    outerRadius,
+                                                    bridgeRadius,
+                                                    regressionG,
+                                                    regressionF,
+                                                    regressionRadius,
+                                                    rzChiSquared,
+                                                    chiSquared,
+                                                    nonAnchorChiSquared,
+                                                    TightCutFlag);
+
+            if (success) {
+              int totOccupancyQuintuplets =
+                  alpaka::atomicOp<alpaka::AtomicAdd>(acc, &quintupletsInGPU.totOccupancyQuintuplets[lowerModule1], 1u);
+              if (totOccupancyQuintuplets >= rangesInGPU.quintupletModuleOccupancy[lowerModule1]) {
+#ifdef Warnings
+                printf("Quintuplet excess alert! Module index = %d\n", lowerModule1);
+#endif
+              } else {
+                int quintupletModuleIndex =
+                    alpaka::atomicOp<alpaka::AtomicAdd>(acc, &quintupletsInGPU.nQuintuplets[lowerModule1], 1u);
+                //this if statement should never get executed!
+                if (rangesInGPU.quintupletModuleIndices[lowerModule1] == -1) {
+#ifdef Warnings
+                  printf("Quintuplets : no memory for module at module index = %d\n", lowerModule1);
+#endif
+                } else {
+                  unsigned int quintupletIndex =
+                      rangesInGPU.quintupletModuleIndices[lowerModule1] + quintupletModuleIndex;
+                  float phi =
+                      mdsInGPU.anchorPhi[segmentsInGPU.mdIndices[2 * tripletsInGPU.segmentIndices[2 * innerTripletIndex +
+                                                                                                  layer2_adjustment]]];
+                  float eta =
+                      mdsInGPU.anchorEta[segmentsInGPU.mdIndices[2 * tripletsInGPU.segmentIndices[2 * innerTripletIndex +
+                                                                                                  layer2_adjustment]]];
+                  float pt = (innerRadius + outerRadius) * 3.8f * 1.602f / (2 * 100 * 5.39f);
```

Comment:
Comment from Matt:
> where are these numbers coming from?

@YonsiG possibly you could help here as well? Also, is this in line with the updates you did recently to the pT of the different objects?

Reply:
Thank you for the very detail explanation, @YonsiG, really helpful!

Please correct me if I am wrong but it seems to me that the expression:
`(innerRadius + outerRadius) * 3.8f * 1.602f / (2 * 100 * 5.39f)`
can just be simplified to:
`(innerRadius + outerRadius) * k2Rinv1GeVf`.

If so, I will change it in the code.

Reply:
Yes, you can do this. The 2 gave very similar numbers, I believe it depends on how many numbers of digit you keep for intermediate numbers:)